core/arch/arm/plat-rzg/conf.mk

```diff
@@ -0,0 +1,47 @@
+PLATFORM_FLAVOR ?= hihope_rzg2m
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_SCIF,y)
+$(call force,CFG_CORE_LARGE_PHYS_ADDR,y)
+$(call force,CFG_CORE_ARM64_PA_BITS,36)
+
+# Disable core ASLR for two reasons:
+# 1. There is no source for ALSR seed, as RZG platform
+#    does not provide DTB to OP-TEE. Also, there is no
+#    publicly available documentation on integrated
+#    hardware RNG, so we can't use it either.
+# 2. OP-TEE crashes during boot with enabled CFG_CORE_ASLR.
+$(call force,CFG_CORE_ASLR,n)
```

Comment:
I see this is a copy/paste from `core/arch/arm/plat-rcar/conf.mk`. Does it really apply here?

In my opinion, the first point is a valid reason for `CFG_CORE_ASLR ?= n`, while the second one would indeed justify `$(call force,CFG_CORE_ASLR,n)` but only if the issue is real and a fix is not found.

Reply:
Yes it does apply on RZ/G SoC's too. (I enabled CFG_CORE_ASLR and did see that u-boot doesnt come up)